scripts/data/common.py

```diff
@@ -0,0 +1,20 @@
+"""Common helpers for data."""
+import sys
+from typing import Any
+import voluptuous as vol
+
+def expand_and_humanize_error(content: dict[str, Any], error: vol.Error) -> list[str] | str:
```

Comment:
`error` is too broadly typed; `vol.Error` is just an `Exception`.
I think we should narrow it to `vol.Invalid`, which has `path` etc., both here and in the scripts which catches voluptuous errors and then calls this function.